.changeset/loud-ties-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Extend Hero heading and description
```

Comment:
Let's call out the importance of checking for hero regressions after this upgrade...


```suggestion
'@primer/react-brand': minor
---

Extend `Hero.Heading` and `Heading.Description` length.

> **Warning**
> This change can lead to reduced contrast in certain situations. Please manually review all instances of the `Hero` - particularly those that use a background image - to ensure that minimum contrast requirements are met. 
```